wiki/Performance_troubleshooting/en.md

```diff
@@ -46,10 +47,12 @@ During gameplay, one or more input devices stop responding. Everything on screen
 During gameplay, the frame rate is unable to keep up, resulting in jittery and awkward movement of objects on the screen. This can also result in higher perceived input latency. When diagnosing or reporting, it may help to:
 
 - Check whether it still happens straight after a PC restart.
+- Check whether it still happens with default or any optimized skin.
 - Try closing any application which shows overlays on the game (Discord, Steam, NVIDIA Shadowplay etc.).
 - Make sure you're not forcing vsync on at your display drivers (it should be set to "off" or "application preference").
 - Try changing frame limiter and see if it improves the situation.
 - Make sure you are running osu! in exclusive (not borderless) fullscreen mode. This will always improve performance.
+  - This may be hindered unintentionally by [fullscreen optimizations](https://devblogs.microsoft.com/directx/demystifying-full-screen-optimizations). Check if disabling it improves performance.
```

Comment:
Do you have any information on this actually helping anywhere? I wouldn't want to add this without such documentation.

Reply:
Please do. Improving "storyboard intensive" sounds very wrong to me. It should not have any effect on intensive scenarios.

Reply:
These results don't help sorry.

I'd need full frame graphs over 5 runs in each mode.